rp_asos/tests/test_views.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib.auth.models import User
+from django.urls import reverse
+from mock import patch
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase, APIClient
+
+from rp_redcap.tests.base import RedcapBaseTestCase
+
+
+class CheckViewTests(RedcapBaseTestCase, APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.client_noauth = APIClient()
+
+        self.username = "testuser"
+        self.password = "testpass"
+        self.user = User.objects.create_user(
+            self.username, "testuser@example.com", self.password
+        )
+        token = Token.objects.create(user=self.user)
+        self.token = token.key
+        self.client.credentials(HTTP_AUTHORIZATION="Token " + self.token)
```

Comment:
```suggestion
        self.client.credentials(HTTP_AUTHORIZATION="Token {}".format(self.token))
```
😬 sorry. nitpicking here